docs-developer/README.md

```diff
@@ -23,18 +23,22 @@ Alternatively, you can use Docker:
 docker run --rm -it -p 8000:8000 -v ./docs-developer/src/doc:/docs squidfunk/mkdocs-material
 ```
 
-This will build the docs and start a web server under [localhost:8000/Kotlin/dokka](http://localhost:8000/Kotlin/dokka/).
+This will build the docs and start a web server under
+[localhost:8000/Kotlin/dokka](http://localhost:8000/Kotlin/dokka/).
 
-### Livereload server
+### Live-reload server
 
-Alternatively, you can run a livereload server that automatically rebuilds documentation on every change:
+If you are using IntelliJ then a link to the docs will be logged in the console:
```

Comment:
If the link is opened using IntelliJ's built in web server, it was surprising for me to find that it is only available in IntelliJ Ultimate, but not in community, so I think it makes sense to clarify it here if that's the case. Also might wanna link to https://www.jetbrains.com/help/idea/editing-html-files.html#ws_html_preview_output

I found out after recommending it to one of Dokka's users, and they spent quite some time looking for how to do it only to realize they don't have that feature :[

Reply:
I thought the built-in server was available in IJ Community, at least the code is there https://github.com/JetBrains/intellij-community/blob/8685843f6ad2a45ef772bf0cdf34590036a1d9e8/platform/built-in-server/src/org/jetbrains/builtInWebServer/BuiltInServerOptions.java

Reply:
I've just installed IJ Community and the built-in server link works.

Reply:
0_o I definitely remember having problems with it in community, and the linked documentation page says "Ultimate" at the top

That's good to know, thanks for checking!